src/app/component/eco-news/components/news-list/news-list.component.ts

```diff
@@ -25,6 +25,7 @@ export class NewsListComponent implements OnInit, OnDestroy {
   private currentPage: number;
   public scroll: boolean;
   public numberOfNews: number;
+  public elementsArePresent = true;
```

Comment:
Why did you use `true` by default?

Reply:
I used true by default for this condition, when we receive the first news on our page

![image](https://user-images.githubusercontent.com/55846203/91437348-6c5a6280-e872-11ea-890d-2cac09a816e5.png)